app/serializers/trucking_audit_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class TruckingAuditSerializer < ActiveModel::Serializer
+  attributes :id, :user_id, :username, :company, :data, :action, :changes, :type
+
+  def company
+    object.company_id&.name
```

Comment:
```suggestion
    object.company&.name
```
company_id  это параметр